admin/dist/mac/bundleFrameworks.sh

```diff
@@ -177,18 +179,18 @@ mkdir -p "$bundlePath/Contents/Resources/qm"
 translations="qt_de.qm
                 qt_es.qm 
                 qt_fr.qm 
-                qt_it.qm 
                 qt_ja.qm 
                 qt_pl.qm 
                 qt_pt.qm 
                 qt_ru.qm 
                 qt_sv.qm 
-                qt_tr.qm
                 qt_zh_CN.qm"
 
 for translation in $translations; do
     if [ -d /Developer/Applications/Qt/plugins/ ]; then
         translationDir=/Developer/Applications/Qt/translations
+    elif [ -d /usr/local/Cellar/qt@4/4.8.7_5/translations/ ]; then
+        translationDir=/usr/local/Cellar/qt@4/4.8.7_5/translations
```

Comment:
This doesn't match the plugindir path (/usr/local/lib/qt4/plugins). Why is that?

Reply:
When installing qt with homebrew, brew symlinks imports and plugins to the local lib. So `/usr/local/lib/qt4/` looks like:
```
drwxr-xr-x imports/
drwxr-xr-x plugins/
```
But it doens't link the translations files anywhere. It just leaves it in the Cellar.

Reply:
Given that we hard-depend on 4.8.7, I think we should make both of them use the direct link, or not use homebrew. Either way it's weird having them mixed

Like, you can't use any other version of QT… so…

Reply:
I can set the script to point to `usr/local/Cellar/qt@4/4.8.7_5/` everywhere qt4 is required then

Reply:
By direct link I mean change the instructions to something like "download QT 4.8.7 and put it in the top level next to Growl, Sparkle…"

Reply:
I don't know. I just feel weird about making this not just a dependency on Qt, but also dependency on Homebrew backports